main.go

```diff
@@ -63,9 +65,21 @@ func main() {
 	// Make logging output better.
 	log.SetFlags(log.Ldate | log.Ltime | log.Lmicroseconds)
 	if strings.ToLower(os.Getenv("ENABLE_GCSFUSE_VIPER_CONFIG")) == "true" {
-		os.Args = convertToPosixArgs(os.Args)
-		cmd.Execute()
+		// TODO: implement the mount logic instead of simply returning nil.
+		rootCmd, err := cmd.NewRootCmd(func(config cfg.Config) error { return nil })
+		if err != nil {
+			exitOnError(err)
```

Comment:
We can use logger.Fatal here too?

Reply:
It seems logger.Fatal writes to stdout instead of stderr. I didn't want to change the failure behavior.

Reply:
When GCSFuse runs in foreground, logs go to stdout, when GCSFuse runs in background, logs go to sys logs and when log file is specified, logs go to log file. I think it is okay to write error log with logger.Fatal (to stdout) in this case as it has the benefit of capturing the log when GCSFuse is run in different mode. WDYT?